IMC_R/EDA_Raymond/contentdevproposal.md

```diff
@@ -1,31 +1,28 @@
-# TEMPLATE
 
-## :fire: Do not edit this file - copy the template and create your own file.
-
-**[Step-By-Step Technical Blog Guide](https://hq.bitproject.org/how-to-write-a-technical-blog/)**
+# EDA for R
 
 ### :pushpin: Step 1
 **TITLE:**    
-{{Insert_title_here}}
+Exploring Movie Streaming Data With R
 
 **TOPIC:**    
-{{Insert_topic_here}}
+Data Analysis with R
 
 **DESCRIPTION (5-7+ sentences):**    
-{{Insert_description_here}}
+Exploratory data analysis, or EDA for short, is an approach to exploring data that can help users obtain basic insights before performing more complex operations. Although the techniques employed by EDA are relatively simple, EDA is a critical component of any statistical analysis. It allows us to understand the underlying structure of the data and evaluate the validity of our assumptions. This blog will help readers understand EDA and employ these techniques in R. It will use the Kaggle movie streaming dataset to illustrate examples of EDA.
 
 ### :pushpin: Step 2
 :family: **TARGET AUDIENCE (3-5+ sentences):**    
-{{Insert_description_here}}
+This blog is targeted towards students who are interested in data analysis and R. It will require relatively little prior knowledge, although readers would benefit from a high school level understanding of statistics, and maybe some familiarity with coding. Experience with R is unnecessary.
 
 ### :pushpin: Step 3
 > Outline your learning/teaching structure: 
 
 **Beginning (2-3+ sentences):**    
-{{Insert_here}}
+The blog will start off with an introduction of EDA. This will include a description of various techniques used for EDA (primarily different types of graphs and tables). Then, I will explain the purpose and significance of EDA.
 
 **Middle (2-3+ sentences):**    
-{{Insert_here}}
+After the introduction, the blog will describe the specific techniques involved in EDA from a statistical point of view. Then, the blog will demonstrate these techniques in R using the tidyverse and/or dataexplorer packages, with the Kaggle movie streaming dataset as an example dataset.
 
 **End (2-3+ sentences):**    
-{{Insert_here}}
+Finally, the blog will conclude with the results from the movie dataset, and a discussion of what those findings tell us in terms of ideal model selection and assumptions.
```

Comment:
be sure to also relate back to the motivation of the intro, purpose of this blog.. etc